pio/ir_nec/README.adoc

```diff
@@ -0,0 +1,57 @@
+= Sending and receiving IR (infra-red) codes using the PIO
+
+This example shows how to use the Raspberry Pi Pico (RP2040) to send and receive infra-red frames in the 'NEC' format that is used by many consumer remote control applications.
+
+It performs a loopback test by transmitting IR codes via an IR LED and receiving them on an IR detector. The results are sent to the default serial terminal connnected via UART or USB as configured in the SDK.
+
+The tasks of encoding and decoding the data are offloaded to the RP2040 PIO state machines. This releases the main processor cores to concentrate on other tasks.
+
+== Wiring information
+
+Connect GPIO 14 to the positive side ('anode') of an IR LED via a suitable series resistor e.g. 1.5k Ohms, and the negative side ('cathode') to ground.
```

Comment:
Does it need to be a specific frequency / wavelength, or will any old random IR LED work?

Reply:
I don't think those detectors are terribly picky so any old one should do (I just scavenged one from an old remote control).
It's a good point however, so I shall amend the wording to the effect that the IR LED should be compatible with the detector